src/content/guides/asset-modules.md

```diff
@@ -26,6 +26,52 @@ Asset Modules type replaces all of these loaders by adding 4 new module types:
 - `asset/source` exports the source code of the asset. Previously achievable by using `raw-loader`.
 - `asset` automatically chooses between exporting a data URI and emitting a separate file. Previously achievable by using `url-loader` with asset size limit.
 
+When using the old assets loaders (i.e. `file-loader`/`url-loader`/`raw-loader`) in webpack 5 you might want to exclude them from the assets module processing to prevent asset duplication. This can be done by overriding the asset configuration for that asset type by setting it's module type to 'javascript/auto'.
```

Comment:
```suggestion
When using the old assets loaders (i.e. `file-loader`/`url-loader`/`raw-loader`) along with Asset Module in webpack 5, you might want to stop Asset Module from processing your assets as that would result in asset duplication. This can be done by setting asset's module type to `'javascript/auto'`.
```